lib/formfields/admin/domains/formfield.domains_edit.php

```diff
@@ -467,6 +489,13 @@
 						'value' => array(
 							$result['dkim']
 						)
+					),
+					'dkiminfo' => array(				// DF8OE
+						'visible' => (\Froxlor\Settings::Get('dkim.use_dkim') == '1' && $result['dkim'] && !$result['isbinddomain'] ? true : false),
+						'visible' => ($result['dkim'] && !$result['isbinddomain'] ? true : false),
+						'label' => 'add to DNS Zonefile:',
+						'type' => 'label',
+						'value' => nl2br($dnsrec).'<br><br>'
```

Comment:
unnecessary newlines at the end of the value, why add to line-breaks there?

Reply:
"It looks nicer with some space here...

Reply:
I see this is dependent on moon-phases, airpressure in Brazil and how much beer last evening :) I have removed the newlines